docs/pages/versions/unversioned/sdk/admob.md

```diff
@@ -50,6 +51,10 @@ await AdMobRewarded.showAdAsync();
 
 ### AdMobBanner
 
+#### servePersonalizedAds property
+
+The default behavior of the Google Mobile Ads SDK is to serve personalized ads. If a user has consented to receive only non-personalized ads, you can configure the view to specify that only non-personalized ads should be requested. Adding `servePersonalizedAds` property causes non-personalized ads to be requested regardless of whether or not the user is in the EEA. The default is `false` — ads won't be personalized.
```

Comment:
It might be confusing that the default behavior of the Google Mobile Ads SDK is to serve personalized ads, but in our wrapper we make `servePersonalizedAds` default to `false` (thereby showing non-personalized ads)
```suggestion
If a user has consented to receive only non-personalized ads, you can configure the view to specify that only non-personalized ads should be requested. Setting the `servePersonalizedAds` property to `true` causes personalized ads to be requested, otherwise non-personalized ads will be requested regardless of whether or not the user is in the EEA. Defaults to `false` — ads won't be personalized.
```